libqtile/widget/net.py

```diff
@@ -29,43 +29,45 @@ class Net(base.ThreadedPollText):
     """Displays interface down and up speed"""
     orientations = base.ORIENTATION_HORIZONTAL
     defaults = [
-        ('interface', 'wlan0', 'The interface to monitor'),
+        ('interface', None, 'List of interfaces to monitor or None to displays all active NICs combined'),
         ('update_interval', 1, 'The update interval.'),
     ]
 
     def __init__(self, **config):
         base.ThreadedPollText.__init__(self, **config)
         self.add_defaults(Net.defaults)
-        self.interfaces = self.get_stats()
+        if not isinstance(self.interface, list):
+            if self.interface is None:
+                self.interface = "all"
+            self.interface = [self.interface]
+        self.stats = self.get_stats()
 
     def convert_b(self, b):
-        # Here we round to 1000 instead of 1024
-        # because of round things
-        letter = 'B'
-        # b is a float, so don't use integer division
-        if int(b / 1000) > 0:
-            b /= 1000.0
-            letter = 'k'
-        if int(b / 1000) > 0:
-            b /= 1000.0
-            letter = 'M'
-        if int(b / 1000) > 0:
-            b /= 1000.0
-            letter = 'G'
-        # I hope no one have more than 999 GB/s
-        return b, letter
 
-    def get_stats(self):
+        letters = ["b", "kb", "Mb", "Gb", "Tb", "Pb", "Eb", "Zb", "Yb"]
```

Comment:
Does this deal in bits or bytes?

Reply:
Bytes must be the obvious answer, though I can't say for sure. So it should be a capitalized B. Another question is: does this deal with multiples of `1000` or `1024` (`2**10`)? Former: `B, kB, MB, GB, TB, PB, EB, ZB, YB`. Latter: `B, KiB, MiB, GiB, TiB, PiB, EiB, ZiB, YiB`

Reply:
As per [psutil.net_io_counters](https://psutil.readthedocs.io/en/latest/#psutil.net_io_counters): it's bytes and not bits.
Currently it's working with multiples of 1000.
I'll have the Bees fixed XD